### ISSUE TITLE
Automatic export does not export when a collection changes

### ISSUE THREAD

**Issue by cossio** [retorquere/zotero-better-bibtex#752]
When I add a new paper to the Library, automatic exports are triggered. But when I move an item from a collection to another collection, automatic exports involving one of the collections affected are not triggered.

**Comment by retorquere** [comment]
Which one is not triggered? The one where the item is move to, or the one where the item is moved out of?

**Comment by cossio** [comment]
Both. When I move an item into a collection, or when I remove an item from a collection, that is supposed to be exported automatically, nothing happens.

**Comment by retorquere** [comment]
I'll need you to generate a debug report, I can't replicate this on my end. When moving references, both collections are triggered for autoexport.

**Comment by retorquere** [comment]
I will really need your help on this as I cannot reproduce it on my end.

**Comment by cossio** [comment]
Now it is working fine. Just like the other issue I created. Once I disabled BBT, and enabled it again, they were both fixed. I am sorry I cannot be of more help. If I find a way to reproduce it consistently I will post here. Let's not rule out the possibility that I might have messed something up or that my system had something wrong :)  I am closing this in the mean time. Thanks for the hard work on this plugin.
